data/latest/test-rules-engine.sh

```diff
@@ -95,35 +116,79 @@ if [ -z "$(edgexfoundry.kuiper-cli create rule rule2 '
          }
       }
    ]
-}' | grep '\bRule rule2 was created successfully\b')" ] ; then
+}' | grep '\bRule rule_mqtt was created successfully\b')" ] ; then
     echo "cannot create kuiper rule (action: mqtt)"
     snap_remove
     exit 1
 fi
 
+# create a rule with action: rule_edgex_message_bus
+if [ -z "$(edgexfoundry.kuiper-cli create rule rule_edgex_message_bus '
+{
+   "sql":"SELECT * from stream1",
+   "actions": [
+      {
+         "edgex": {
+            "connectionSelector": "edgex.redisMsgBus",
+            "topicPrefix": "edgex/events/device", 
+            "messageType": "request",
+            "deviceName": "device-test"
+         }
+      }
+   ]
+}' | grep '\bRule rule_edgex_message_bus was created successfully\b')" ] ; then
```

Comment:
Suggestion: Move the rule out of the if statement and store it in a variable. Replace the string inline to make the condition shorter.

I understand this is consistent with the rest of the code. But it would be a good start to improve the way such conditions are written so that the `then` keyword isn't so away from `if`. This will improve writability and readability.

Reply:
I agree with you. Updated.